src/app/feed/feed-header/feed-header.component.scss

```diff
@@ -10,7 +10,7 @@
 		flex-wrap: wrap;
 		align-items: center;
 		width: 100%;
-		height: 85px;
+		height: 155px;
```

Comment:
@rishiraj824 To get menubar just like the market leader. Height needs to be changed. 
Any design idea?